Desktop/data/datasetpackage.cpp

```diff
@@ -205,22 +267,26 @@ int DataSetPackage::rowCount(const QModelIndex & parent) const
 		int labelSize = col.labels().size();
 		return labelSize;
 	}
-	case parIdxType::filter:
-	case parIdxType::root:		//return int(parIdxType::leaf); Its more logical to get the actual datasize
-	case parIdxType::data:		return !_dataSet ? 0 : _dataSet->rowCount();
+	case parIdxType::filterRoot:	[[clang::fallthrough]];
```

Comment:
I think you can just use `[[fallthrough]]`, no?

Reply:
I can definitely try, that would look a lot better.

Reply:
That could very well be. Full C++17 is not very new, especially on GCC.

Reply:
Haha, no I am aware we are living in the future, but it took us a while to switch to >11 even.